src/libcollections/btree/map.rs

```diff
@@ -326,6 +326,15 @@ pub enum Entry<'a, K: 'a, V: 'a> {
              OccupiedEntry<'a, K, V>),
 }
 
+impl<'a, K: 'a + Debug + Ord, V: 'a + Debug> Debug for Entry<'a, K, V> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            &Vacant(ref v) => write!(f, "Entry({:?})", v),
```

Comment:
It would be nice if these took into account the formatting parameters. You can probably use [`Formatter::debug_tuple`](https://doc.rust-lang.org/nightly/std/fmt/struct.Formatter.html#method.debug_tuple) for all of these.